src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -579,20 +587,25 @@ public void TotalProcessorTime_GetNotStarted_ThrowsInvalidOperationException()
         }
 
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/corefx/issues/22266", TargetFrameworkMonikers.UapNotUapAot)]
         public void TestProcessStartTime()
         {
-            TimeSpan allowedWindow = TimeSpan.FromSeconds(3);
-            DateTime testStartTime = DateTime.UtcNow;
-            using (var remote = RemoteInvoke(() => { Console.Write(Process.GetCurrentProcess().StartTime.ToUniversalTime()); return SuccessExitCode; },
-                new RemoteInvokeOptions { StartInfo = new ProcessStartInfo { RedirectStandardOutput = true } }))
-            {
-                Assert.Equal(remote.Process.StartTime, remote.Process.StartTime);
+            TimeSpan allowedWindow = TimeSpan.FromSeconds(1);
```

Comment:
This will be broken by clock adjustments, it needs the same protection I did for Process ExitTime, see 925417b1a65fbee91cf3a1c4b84b06045c9eb0e8.